compiler/platform.nim

```diff
@@ -147,6 +147,11 @@ const
       objExt: ".o", newLine: "\x0A", pathSep: ";", dirSep: "\\",
       scriptExt: ".sh", curDir: ".", exeExt: ".vxe", extSep: ".",
       props: {ospNeedsPIC, ospPosix, ospLacksThreadVars}),
+     (name: "Genode", pardir: "..", dllFrmt: "$1.lib.so", altDirSep: "/",
+      objExt: ".o", newLine: "\x0A", pathSep: ":", dirSep: "/",
+      scriptExt: "", curDir: "/", exeExt: "", extSep: ".",
```

Comment:
@ehmry Is '/' really used as current directory in Genode?
It looks like error as it is same to `dirSep`.

Reply:
`curDir` could have probably just been `./` but Genode doesn't have the concept of "current working directory" unless you use a Libc, in which case you have standard POSIX behavior. Without a Libc any relative path will be rooted at the root of the file-system namepace that you are working in.

Reply:
`CurDir` is also defined to `'/'` here:
https://github.com/nim-lang/Nim/blob/devel/lib/std/private/osseps.nim#L14
It is also same to `DirSep`.

Reply:
It's awkward but it hasn't been an issue so far.